commands/create_cluster.go

```diff
@@ -416,8 +416,13 @@ func definitionFromFlags(def clusterDefinition, args addClusterArguments) cluste
 	}
 
 	if def.Scaling.Min == 0 && def.Scaling.Max == 0 {
-		def.Scaling.Min = int64(len(def.Workers))
-		def.Scaling.Max = int64(len(def.Workers))
+		def.Scaling.Min = int64(args.numWorkers)
```

Comment:
`def.Workers` is not set here yet, this solves a problem where scaling values were always `0` if only `num-workers`  was given.